python/hail/expr/expression.py

```diff
@@ -1221,7 +1292,7 @@ def sort(self, ascending=True):
 
         Parameters
         ----------
-        ascending : bool or :py:class:`BooleanExpression`
+        ascending : :py:class:`BooleanExpression`
```

Comment:
There are a bunch of references that won't link. Missing ".". I propose we fix these in another PR.

Reply:
yeah, this is a problem everywhere.